<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <title>Publications - Dexiong Chen</title>
    <meta property="og:title" content="Publications - Dexiong Chen">
    
    <meta name="twitter:card" content="summary">

    
      
    

    
      
    

    
    

    

    
    

    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.1/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous" />
    
  </head>

  
  <body class="publications">
    <header class="masthead">
      <h1><a href="/">Dexiong Chen</a></h1>



<p class="contact"><i class="fas fa-envelope-square"></i> dexiong.chen@bsse.ethz.ch</p>

<p class="contact"><a href="https://github.com/claying"><i class='fab fa-github-square'></i></a> <a href="https://gitlab.inria.fr/dchen"><i class='fab fa-gitlab'></i></a></p>


      <nav class="menu">
  <input id="menu-check" type="checkbox" hidden/>
  <label id="menu-label" for="menu-check" class="unselectable" hidden>
    <span class="icon close-icon">✕</span>
    <span class="icon open-icon">☰</span>
    <span class="text">Menu</span>
  </label>
  <ul>
  
  
  <li class="active"><a href="/publications/">Publications</a></li>
  
  <li><a href="/software/">Software</a></li>
  
  <li><a href="/teaching/">Teaching</a></li>
  
  <li><a href="/talks/">Talks</a></li>
  
  
  </ul>
</nav>

    </header>

    <article class="main">
      <header class="title">
      <h1>Publications</h1>

<h3>
</h3>
<hr>

      </header>



<ul>
<li><a href="https://arxiv.org/abs/2207.02968"><strong>Unsupervised Manifold Alignment with Joint Multidimensional Scaling</strong></a> <br />
<strong>Dexiong Chen</strong>, Bowen Fan, Carlos Oliver, Karsten Borgwardt <br />
<em>Preprint</em>, 2022 <br />
<a href="https://arxiv.org/abs/2207.02968"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/JointMDS"><i class="fas fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2206.01008"><strong>Approximate Network Motif Mining via Graph Learning</strong></a> <br />
Carlos Oliver, <strong>Dexiong Chen</strong>, Vincent Mallet, Pericles Philippopoulos, Karsten Borgwardt <br />
<em>Preprint</em>, 2022 <br />
<a href="https://arxiv.org/abs/2206.01008"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/MotiFiesta"><i class="fas fa-file-code"></i></a></li>
<li><a href="https://www.frontiersin.org/articles/10.3389/fninf.2022.1032538/abstract"><strong>Predicting in vitro single-neuron firing rates upon pharmacological perturbation using graph neural networks</strong></a> <br />
Taehoon Kim, <strong>Dexiong Chen</strong>, Philipp Hornauer, Vishalini Emmenegger, Julian Bartram, Silvia Ronchi, Andreas R. Hierlemann, Manuel Schröter and Damian Roqueiro <br />
<em>Frontiers in Neuroinformatics</em>, 2022 <br />
<a href="https://www.frontiersin.org/articles/10.3389/fninf.2022.1032538/abstract"><i class="fas fa-file-pdf"></i></a></li>
<li><a href="https://arxiv.org/abs/2202.03036"><strong>Structure-Aware Transformer for Graph Representation Learning</strong></a> <br />
<strong>Dexiong Chen</strong>, Leslie O&rsquo;Bray, Karsten Borgwardt <br />
<em>ICML</em>, 2022 <br />
<a href="https://arxiv.org/abs/2202.03036"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/SAT"><i class="fas fa-file-code"></i></a> <a href="poster/icml2022.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/2106.05667"><strong>GraphiT: Encoding Graph Structure in Transformers</strong></a> <br />
Grégoire Mialon, <strong>Dexiong Chen</strong>, Margot Selosse, Julien Mairal <br />
<em>Preprint</em>, 2021 <br />
<a href="https://arxiv.org/abs/2106.05667"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/inria-thoth/GraphiT"><i class="fas fa-file-code"></i></a></li>
<li><a href="https://hal.inria.fr/tel-03193220"><strong>Structured Data Modeling with Deep Kernel Machines and Applications in Computational Biology</strong></a> <br />
<strong>Dexiong Chen</strong> <br />
PhD thesis, Université Grenoble Alpes, 2020 <br />
<a href="https://hal.inria.fr/tel-03193220"><i class="fas fa-file-pdf"></i></a></li>
<li><a href="https://arxiv.org/abs/2006.12065"><strong>A Trainable Optimal Transport Embedding for Feature Aggregation and its Relationship to Attention</strong></a> <br />
Grégoire Mialon, <strong>Dexiong Chen</strong>, Alexandre d&rsquo;Aspremont, Julien Mairal <br />
<em>ICLR</em>, 2021 <br />
<a href="https://arxiv.org/abs/2006.12065"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/claying/OTK"><i class="fas fa-file-code"></i></a> <a href="poster/iclr2021.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/2003.05189"><strong>Convolutional Kernel Networks for Graph-Structured Data</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
In <em>ICML</em>, 2020 <br />
<a href="https://arxiv.org/abs/2003.05189"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/claying/GCKN"><i class="fas fa-file-code"></i></a> <a href="https://icml.cc/media/Slides/icml/2020/virtual(no-parent)-16-20-00UTC-6029-convolutional_k.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/1906.03200"><strong>Recurrent Kernel Networks</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
In <em>NeurIPS</em>, 2019 and accepted as an oral presentation in <em>MLCB</em>, 2019 <br />
<a href="https://arxiv.org/abs/1906.03200"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/claying/RKN"><i class="fas fa-file-code"></i></a> <a href="poster/neurips2019.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/1810.00363"><strong>A Kernel Perspective for Regularizing Deep Neural Networks</strong></a> <br />
Alberto Bietti, Grégoire Mialon, <strong>Dexiong Chen</strong>, Julien Mairal <br />
In <em>ICML</em>, 2019 <br />
<a href="https://arxiv.org/abs/1810.00363"><i class="fas fa-file-pdf"></i></a> <a href="https://github.com/albietz/kernel_reg"><i class="fas fa-file-code"></i></a> <a href="poster/icml2019.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
<li><a href="https://doi.org/10.1101/217257"><strong>Biological Sequence Modeling with Convolutional Kernel Networks</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
In <em>Research in Computational Molecular Biology (RECOMB)</em>, 2019 and <em>Bioinformatics</em>, 2019 <br />
<a href="https://doi.org/10.1101/217257"><i class="fas fa-file-pdf"></i></a> <a href="https://gitlab.inria.fr/dchen/CKN-seq"><i class="fas fa-file-code"></i></a> <a href="poster/mlss2019.pdf"><i class="fas fa-file-powerpoint"></i></a></li>
</ul>


  <footer>
  
  

  
  <hr>
  <div class="copyright">© Dexiong Chen 2019-2022</div>
  
  </footer>
  </article>
  
  </body>
</html>

