<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <title>Publications - Dexiong Chen</title>
    <meta property="og:title" content="Publications - Dexiong Chen">
    
    <meta name="twitter:card" content="summary">

    
      
    

    
      
    

    
    

    

    
    

    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jpswalsh/academicons@1/css/academicons.min.css" />
    <script src="https://kit.fontawesome.com/64564216b5.js" crossorigin="anonymous"></script>
    
      <link rel="stylesheet" href="/css/main.min.d93b80e9a82afefd7f1ef17cd48497caf7ae79fc478d98f2ebd8b12400733721.css" integrity="sha256-2TuA6agq/v1/HvF81ISXyveuefxHjZjy69ixJABzNyE=" crossorigin="anonymous">
  </head>

  
  <body class="publications">
    <header class="masthead">
      <h1><a href="/">Dexiong Chen</a></h1>



<p class="contact"><i class="fa-regular fa-envelope"></i> dchen@biochem.mpg.de</p>

<p class="contact"><a href="https://scholar.google.com/citations?user=goM0yAIAAAAJ"><i class='ai ai-google-scholar-square ai-2x'></i></a> <a href="https://github.com/claying"><i class='fa-brands fa-github-square fa-2x'></i></a> <a href="https://www.linkedin.com/in/dexiong-chen-12206aa5/"><i class='fa-brands fa-linkedin fa-2x'></i></a> <a href="https://gitlab.inria.fr/dchen"><i class='fa-brands fa-gitlab-square fa-2x'></i></a></p>


      <nav class="menu">
  <input id="menu-check" type="checkbox" hidden/>
  <label id="menu-label" for="menu-check" class="unselectable" hidden>
    <span class="icon close-icon">✕</span>
    <span class="icon open-icon">☰</span>
    <span class="text">Menu</span>
  </label>
  <ul>
  
  
  <li class="active"><a href="/publications/">Publications</a></li>
  
  <li><a href="/software/">Software</a></li>
  
  <li><a href="/team/">Team</a></li>
  
  <li><a href="/teaching/">Teaching</a></li>
  
  <li><a href="/talks/">Talks</a></li>
  
  <li><a href="/blogs/">Blogs</a></li>
  
  
  </ul>
</nav>

    </header>

    <article class="main">
      <header class="title">
      <h1>Publications</h1>

<h3>
</h3>
<hr>



      </header>



<ul>
<li><a href="https://arxiv.org/abs/2510.06122"><strong>PolyGraph Discrepancy: a classifier-based metric for graph generation</strong></a> <br />
Markus Krimmel, Philip Hartout, Karsten Borgwardt, <strong>Dexiong Chen</strong> <br />
<em>Preprint</em>, 2025 <br />
<a href="https://arxiv.org/abs/2510.06122"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/polygraph-benchmark"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2502.02415"><strong>Towards Fast Graph Generation via Autoregressive Noisy Filtration Modeling</strong></a> <br />
Markus Krimmel, Jenna Wiens, Karsten Borgwardt, Dexiong Chen <br />
<em>Preprint</em>, 2025 <br />
<a href="https://arxiv.org/abs/2502.02415"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/anfm"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2401.14819"><strong>Endowing Protein Language Models with Structural Knowledge</strong></a> <br />
<strong>Dexiong Chen</strong>, Philip Hartout, Paolo Pellizzoni, Carlos Oliver, Karsten Borgwardt <br />
<em>Preprint</em>, 2024 <br />
<a href="https://arxiv.org/abs/2401.14819"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/PST"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2502.02216"><strong>Flatten Graphs as Sequences: Transformers are Scalable Graph Generators</strong></a> <br />
<strong>Dexiong Chen</strong>, Markus Krimmel, Karsten Borgwardt <br />
<em>NeurIPS</em>, 2025 <br />
<a href="https://arxiv.org/abs/2502.02216"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/AutoGraph"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2504.20430"><strong>Learning Laplacian Positional Encodings for Heterophilous Graphs</strong></a> <br />
Michael Ito, Jiong Zhu, <strong>Dexiong Chen</strong>, Danai Koutra, Jenna Wiens <br />
<em>AISTATS</em>, 2025 <br />
<a href="https://arxiv.org/abs/2504.20430"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/MLD3/LearningLaplacianPEs"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2406.03386"><strong>Learning Long Range Dependencies on Graphs via Random Walks</strong></a> <br />
<strong>Dexiong Chen</strong>, Till Hendrik Schulz, Karsten Borgwardt <br />
<em>ICLR</em>, 2025 <br />
<a href="https://arxiv.org/abs/2406.03386"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/NeuralWalker"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://link.springer.com/chapter/10.1007/978-3-031-90252-9_5"><strong>Detecting Antimicrobial Resistance Through MALDI-TOF Mass Spectrometry with Statistical Guarantees Using Conformal Prediction</strong></a> <br />
Nina Corvelo Benz, Lucas Miranda, Dexiong Chen, Janko Sattler, Karsten Borgwardt <br />
<em>RECOMB</em>, 2025 <br />
<a href="https://link.springer.com/chapter/10.1007/978-3-031-90252-9_5"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/ConformalAMR"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0031320324007180"><strong>HTR-VT: Handwritten Text Recognition with Vision Transformer</strong></a>  <br />
Yuting Li, <strong>Dexiong Chen</strong>, Tinglong Tang, Xi Shen <br />
<em>Pattern Recognition</em>, 2025 <br />
<a href="https://www.sciencedirect.com/science/article/pii/S0031320324007180"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://yutingli0606.github.io/HTR-VT/"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://openreview.net/forum?id=8APPypS0yN"><strong>On the Expressivity and Sample Complexity of Node-Individualized Graph Neural Networks</strong></a> <br />
Paolo Pellizzoni, Till Hendrik Schulz, <strong>Dexiong Chen</strong>, Karsten Borgwardt <br />
<em>NeurIPS</em>, 2024 <br />
<a href="https://openreview.net/forum?id=8APPypS0yN"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/NodeIndividualizedGNNs"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://academic.oup.com/bioinformatics/article/40/Supplement_1/i501/7700883"><strong>Biomarker Identification by Interpretable Maximum Mean Discrepancy</strong></a> <br />
Michael F Adamer, Sarah C Brüningk, <strong>Dexiong Chen</strong>, Karsten Borgwardt <br />
<em>ISMB</em>, 2024 <br />
<a href="https://academic.oup.com/bioinformatics/article/40/Supplement_1/i501/7700883"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/spinoptmmd"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://openaccess.thecvf.com/content/CVPR2024/papers/Li_SURE_SUrvey_REcipes_for_building_reliable_and_robust_deep_networks_CVPR_2024_paper.pdf"><strong>SURE: SUrvey REcipes for building reliable and robust deep networks</strong></a> <br />
Yuting Li, Yingyi Chen, Xuanlong Yu, <strong>Dexiong Chen</strong>, Xi Shen <br />
<em>CVPR</em>, 2024 <br />
<a href="https://openaccess.thecvf.com/content/CVPR2024/papers/Li_SURE_SUrvey_REcipes_for_building_reliable_and_robust_deep_networks_CVPR_2024_paper.pdf"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/YutingLi0606/SURE"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://openreview.net/forum?id=27vPcG4vKV"><strong>ProteinShake: Building Datasets and Benchmarks for Deep Learning on Protein Structures</strong></a> <br />
Tim Kucera, Carlos Oliver, <strong>Dexiong Chen</strong>, Karsten Borgwardt <br />
<em>NeurIPS (Datasets and Benchmarks)</em>, 2023 <br />
<a href="https://openreview.net/forum?id=27vPcG4vKV"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://proteinshake.ai/"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://arxiv.org/abs/2305.07580"><strong>Fisher Information Embedding for Node and Graph Learning</strong></a> <br />
<strong>Dexiong Chen</strong>, Paolo Pellizzoni, Karsten Borgwardt <br />
<em>ICML</em>, 2023 <br />
<a href="https://arxiv.org/abs/2305.07580"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/fisher_information_embedding"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://openreview.net/forum?id=lUpjsrKItz4"><strong>Unsupervised Manifold Alignment with Joint Multidimensional Scaling</strong></a> <br />
<strong>Dexiong Chen</strong>, Bowen Fan, Carlos Oliver, Karsten Borgwardt <br />
<em>ICLR</em>, 2023 <br />
<a href="https://arxiv.org/abs/2207.02968"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/JointMDS"><i class="fa-regular fa-file-code"></i></a> <a href="poster/iclr2023.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/2206.01008"><strong>Approximate Network Motif Mining via Graph Learning</strong></a> <br />
Carlos Oliver, <strong>Dexiong Chen</strong>, Vincent Mallet, Pericles Philippopoulos, Karsten Borgwardt <br />
<em>Preprint</em>, 2022 <br />
<a href="https://arxiv.org/abs/2206.01008"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/MotiFiesta"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://www.frontiersin.org/articles/10.3389/fninf.2022.1032538/abstract"><strong>Predicting in vitro single-neuron firing rates upon pharmacological perturbation using graph neural networks</strong></a> <br />
Taehoon Kim, <strong>Dexiong Chen</strong>, Philipp Hornauer, Vishalini Emmenegger, Julian Bartram, Silvia Ronchi, Andreas R. Hierlemann, Manuel Schröter and Damian Roqueiro <br />
<em>Frontiers in Neuroinformatics</em>, 2022 <br />
<a href="https://www.frontiersin.org/articles/10.3389/fninf.2022.1032538/abstract"><i class="fa-regular fa-file-pdf"></i></a></li>
<li><a href="https://arxiv.org/abs/2202.03036"><strong>Structure-Aware Transformer for Graph Representation Learning</strong></a> <br />
<strong>Dexiong Chen</strong>, Leslie O&rsquo;Bray, Karsten Borgwardt <br />
<em>ICML</em>, 2022 <br />
<a href="https://arxiv.org/abs/2202.03036"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/BorgwardtLab/SAT"><i class="fa-regular fa-file-code"></i></a> <a href="poster/icml2022.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/2106.05667"><strong>GraphiT: Encoding Graph Structure in Transformers</strong></a> <br />
Grégoire Mialon, <strong>Dexiong Chen</strong>, Margot Selosse, Julien Mairal <br />
<em>Preprint</em>, 2021 <br />
<a href="https://arxiv.org/abs/2106.05667"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/inria-thoth/GraphiT"><i class="fa-regular fa-file-code"></i></a></li>
<li><a href="https://ieeexplore.ieee.org/abstract/document/9366422"><strong>Metamixup: Learning Adaptive Interpolation Policy of Mixup with Metalearning</strong></a> <br />
Zhijun Mai, Guosheng Hu, <strong>Dexiong Chen</strong>, Fumin Shen, Heng Tao Shen <br />
<em>TNNLS</em>, 2021 <br /></li>
<li><a href="https://hal.inria.fr/tel-03193220"><strong>Structured Data Modeling with Deep Kernel Machines and Applications in Computational Biology</strong></a> <br />
<strong>Dexiong Chen</strong> <br />
PhD thesis, Université Grenoble Alpes, 2020 <br />
<a href="https://hal.inria.fr/tel-03193220"><i class="fa-regular fa-file-pdf"></i></a></li>
<li><a href="https://arxiv.org/abs/2006.12065"><strong>A Trainable Optimal Transport Embedding for Feature Aggregation and its Relationship to Attention</strong></a> <br />
Grégoire Mialon, <strong>Dexiong Chen</strong>, Alexandre d&rsquo;Aspremont, Julien Mairal <br />
<em>ICLR</em>, 2021 <br />
<a href="https://arxiv.org/abs/2006.12065"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/claying/OTK"><i class="fa-regular fa-file-code"></i></a> <a href="poster/iclr2021.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/2003.05189"><strong>Convolutional Kernel Networks for Graph-Structured Data</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
In <em>ICML</em>, 2020 <br />
<a href="https://arxiv.org/abs/2003.05189"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/claying/GCKN"><i class="fa-regular fa-file-code"></i></a> <a href="https://icml.cc/media/Slides/icml/2020/virtual(no-parent)-16-20-00UTC-6029-convolutional_k.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/1906.03200"><strong>Recurrent Kernel Networks</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
<em>NeurIPS</em>, 2019 and accepted as an oral presentation in <em>MLCB</em>, 2019 <br />
<a href="https://arxiv.org/abs/1906.03200"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/claying/RKN"><i class="fa-regular fa-file-code"></i></a> <a href="poster/neurips2019.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://arxiv.org/abs/1810.00363"><strong>A Kernel Perspective for Regularizing Deep Neural Networks</strong></a> <br />
Alberto Bietti, Grégoire Mialon, <strong>Dexiong Chen</strong>, Julien Mairal <br />
<em>ICML</em>, 2019 <br />
<a href="https://arxiv.org/abs/1810.00363"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://github.com/albietz/kernel_reg"><i class="fa-regular fa-file-code"></i></a> <a href="poster/icml2019.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
<li><a href="https://doi.org/10.1101/217257"><strong>Biological Sequence Modeling with Convolutional Kernel Networks</strong></a> <br />
<strong>Dexiong Chen</strong>, Laurent Jacob, Julien Mairal <br />
<em>Research in Computational Molecular Biology (RECOMB)</em>, 2019 and <em>Bioinformatics</em>, 2019 <br />
<a href="https://doi.org/10.1101/217257"><i class="fa-regular fa-file-pdf"></i></a> <a href="https://gitlab.inria.fr/dchen/CKN-seq"><i class="fa-regular fa-file-code"></i></a> <a href="poster/mlss2019.pdf"><i class="fa-regular fa-file-powerpoint"></i></a></li>
</ul>


  <footer>
  


  

  
  <hr>
  <div class="copyright">© Dexiong Chen 2019-2025</div>
  
  </footer>
  </article>
  
      <script async src="https://www.googletagmanager.com/gtag/js?id=G-096L0VPDR9"></script>
      <script>
        var doNotTrack = false;
        if ( false ) {
          var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
          var doNotTrack = (dnt == "1" || dnt == "yes");
        }
        if (!doNotTrack) {
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());
          gtag('config', 'G-096L0VPDR9');
        }
      </script>
  </body>
</html>

